Jenkinsfile

```diff
@@ -1053,6 +1053,11 @@ def dumpFilteredEnvironment(){
   echo "DOCKER_CACHE: ${env.DOCKER_CACHE}"
   echo "GOPACKAGES_COMMA_SEP: ${env.GOPACKAGES_COMMA_SEP}"
   echo "PIP_INSTALL_PARAMS: ${env.PIP_INSTALL_PARAMS}"
+  echo(env.getEnvironment()
+          .findAll{ it.key.startsWith('BUILD_') }
+          .collect(
+            {"${it.key} = ${it.value}"}
+          ).join("\n"))
```

Comment:
I'm afraid this is not supported by default unless the script is granted
```suggestion
```

I'll delete this section for the time being and we will enable some easier to debug when rebuilding the pipeline.

```

  echo(env.getEnvironment()
          .findAll{ it.key.startsWith('BUILD_') }
          .collect(
            {"${it.key} = ${it.value}"}
          ).join("\n"))
```